services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -444,6 +444,26 @@ module.exports = {
       return isArray(sectionQuery) ? sectionQuery : [];
     },
 
+    /**
+     * Return the oldest start date for the current section
+     */
+    sectionStartDate: async (content, { input }, { basedb }) => {
+      const { sectionQuery } = content;
+      const { sectionId } = input;
+      const descendantIds = await getDescendantIds(sectionId, basedb);
+      const sectionSchedules = sectionQuery.filter(section => section.sectionId === sectionId);
+      const descendantSectionSchedules = sectionQuery
+        .filter(section => descendantIds.includes(section.sectionId))
+        .map(section => section.start);
```

Comment:
I wasn't explicit about it in my review, but combine these. There's no need to create two arrays just to join them again later. The website section ID is always included in its own descendant ids, but you can also just include it in your ids to check.
```js
const ids = [
  sectionId,
  ...(useSectionBubbling ? await getDescendantIds(...) : []),
];